tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -0,0 +1,540 @@
+  will continue to be supported for some time after 4.0 is released, but it will
+  eventually stop receiving support.
+
+Disadvantages of upgrading
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't *need* any features present in Godot 4.0, you may want to stay on
+Godot 3.x for the following reasons:
+
+- Godot 4's baseline hardware requirements (such as memory usage) are slightly
+  higher, both for the editor and exported projects. This was required for the
+  implementation of some core optimizations.
+- Godot 4 does not and will not have support for GLES2 rendering.
+  (There is still support for GLES3 rendering using the new OpenGL backend,
+  which means that devices without Vulkan support can still run Godot 4.)
+
+  - If you are targeting **very** old hardware such as Intel Sandy Bridge (2nd
+    generation) integrated graphics, this will prevent the project from running
+    on such hardware after upgrading.
+    `Software OpenGL implementations <https://github.com/pal1000/mesa-dist-win>`__
+    can be used to bypass this limitation, but they're too slow for gaming.
+
+Running the project upgrade tool
+--------------------------------
+
+.. warning::
+
+    **Make a full backup of your project** before upgrading! The project upgrade
+    tool will *not* perform any backups of the project that is being upgraded.
+
+    You can backup a project by using version control, or by copying the project
+    folder to another location.
+
+To use the project upgrade tool:
+
+1. Open the Godot 4 project manager.
+2. Import the Godot 3.x project using the **Import** button, or use the **Scan**
+   button to find the project within a folder.
+3. Double-click the imported project (or select the project then choose **Edit**).
+4. You will see a dialog appearing with two options: **Convert project.godot
+   Only** and **Convert Full Project**. After ensuring your project is backed up
+   (see the above warning), choose **Convert Full Project**. **Convert
+   project.godot Only** is intended to be used for advanced use cases *only*, in
+   case the conversion tool fails.
+5. Wait until the project conversion process finishes. This can take up to a few
+   minutes for large projects with lots of scenes.
+6. When the project manager interface becomes available again, double-click the
+   project (or select the project then choose **Edit**) to open it in the
+   editor.
+
+.. note::
+
+    Only Godot 3.0 and later projects can be upgraded using the project
+    conversion tool found in the Godot 4 editor.
+
+    Projects made using Godot versions older than 3.0 must follow a 2-step
+    process:
+
+    1. Upgrade the project to Godot 3 using the **Tools > Export to Godot 3.0**
+       option in the Godot 2.1.6 editor. Open the upgraded project in the Godot
+       3.x editor, then manually fix the project to make sure it looks and
+       runs correctly.
+    2. Only *after* performing step 1, you may upgrade the project to Godot 4.
```

Comment:
I'm not sure this is worth describing, it literally affects maybe 2 users, who know well what they would have to do to convert to 3.x if wanted.